pipeline/foraging_analysis.py

```diff
@@ -39,7 +39,7 @@ def make(self, key):
         trial_stats['double_dipping'] = len(np.unique(q_all_licks_after_go_cue.fetch('action_event_type'))) > 1
         
 
-        self.insert1({**key, **trial_stats})
+        self.insert1({**key, **trial_stats}, skip_duplicates=True )
```

Comment:
why the need for `skip_duplicates=True` here?

Reply:
Oh, that's just a workaround for parallel populating with my local PC on the `map_v2_` database. It seemed that I cannot use `populate(reserve_jobs=True)` on `map_v2_`, and I got the following error even if I called `schema.jobs`. So I had to use `populate(reserve_jobs=False)` with `skip_duplicates=True` to do parallel populating. This change is temporary and should indeed not be merged in the main pipeline... 
```
>>> schema = dj.schema(get_schema_name('foraging_analysis'))
>>> schema.jobs
Traceback (most recent call last):

  File "C:\ProgramData\Anaconda3\envs\map_pipeline\lib\site-packages\IPython\core\formatters.py", line 702, in __call__
    printer.pretty(obj)

  File "C:\ProgramData\Anaconda3\envs\map_pipeline\lib\site-packages\IPython\lib\pretty.py", line 394, in pretty
    return _repr_pprint(obj, self, cycle)

  File "C:\ProgramData\Anaconda3\envs\map_pipeline\lib\site-packages\IPython\lib\pretty.py", line 700, in _repr_pprint
    output = repr(obj)

  File "C:\ProgramData\Anaconda3\envs\map_pipeline\lib\site-packages\datajoint\expression.py", line 395, in __repr__
    return super().__repr__() if config['loglevel'].lower() == 'debug' else self.preview()

  File "C:\ProgramData\Anaconda3\envs\map_pipeline\lib\site-packages\datajoint\expression.py", line 401, in preview
    heading = self.heading

  File "C:\ProgramData\Anaconda3\envs\map_pipeline\lib\site-packages\datajoint\table.py", line 50, in heading
    self.connection, self.database, self.table_name, self.declaration_context)

  File "C:\ProgramData\Anaconda3\envs\map_pipeline\lib\site-packages\datajoint\heading.py", line 162, in init_from_database
    table_name=table_name, database=database))

DataJointError: The table `map_v2_foraging_analysis`.`~jobs` is not defined.

Traceback (most recent call last):

  File "C:\ProgramData\Anaconda3\envs\map_pipeline\lib\site-packages\IPython\core\formatters.py", line 345, in __call__
    return method()

  File "C:\ProgramData\Anaconda3\envs\map_pipeline\lib\site-packages\datajoint\expression.py", line 424, in _repr_html_
    heading = self.heading

  File "C:\ProgramData\Anaconda3\envs\map_pipeline\lib\site-packages\datajoint\table.py", line 50, in heading
    self.connection, self.database, self.table_name, self.declaration_context)

  File "C:\ProgramData\Anaconda3\envs\map_pipeline\lib\site-packages\datajoint\heading.py", line 162, in init_from_database
    table_name=table_name, database=database))

DataJointError: The table `map_v2_foraging_analysis`.`~jobs` is not defined.```